imod/mf6/utilities/clip.py

```diff
@@ -54,9 +54,13 @@ def clip_by_grid(
     package: IPointDataPackage, active: xu.UgridDataArray
 ) -> IPointDataPackage:
     """Clip PointDataPackage outside unstructured grid."""
+
```

Comment:
in many tests, "active" is taken as the top layer of idomain; and the top layer of idomain may sometimes contain some vertical passthrough (vpt) cells, even if the cells below it are active. Should wells coinciding with vpt cells stay or go? Or should we raise an exception if active contains vpt cells? Or should we make a function to infer active from idomain, and only assign vpt if all layers are vpt in idomain for a particular cell?

Reply:
Good point. I think we need to decide what should be 3D and what should be 2D, I'll create a separate issue for this.